index.html

```diff
@@ -5,9 +5,374 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link href="./style.css" rel="stylesheet">
-    <title>NyTimes copy</title>
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun - The New York Times</title>
 </head>
 <body>
-    
+    <nav class="nav-bar">
+        <div class="nav-icon-container">
+            <img src="./resources/left-icons.png" alt="left side icons" class="nav-icon">
+        </div>
+        <div class="nav-logo-container">
+            <img src="./resources/ny-times-logo.png" alt="nytimes logo" class="nav-logo">
+        </div>
+        <div class="nav-btn-container">
+            <button class="nav-btn">Log in</button>
+        </div>   
+    </nav>
+    <div class="header-container">
+        <header>
+            <h1 class="title-container">
+                <span class="title"> 
+                    Space Ripples Reveal <br>Big Bang’s Smoking Gun
+                </span>
+            </h1>
+            <img class="header-img" src="./resources/18cosmos-superjumbo.jpg" alt="main image with telescope">
+            <figcaption class="header-img-figcaption">
```

Comment:
Great work using the figcaption tag. This improves the semantic of your code. But you need to use the figure tag as well. Put the img tag inside a figure tag.